Source/Core/Common/TraversalProto.h

```diff
@@ -1,44 +1,43 @@
 // This file is public domain, in case it's useful to anyone. -comex
 
 #pragma once
+
 #include <array>
+#include <cstddef>
 #include "Common/CommonTypes.h"
 
-#define NETPLAY_CODE_SIZE 8
-typedef std::array<char, NETPLAY_CODE_SIZE> TraversalHostId;
-typedef u64 TraversalRequestId;
+constexpr size_t NETPLAY_CODE_SIZE = 8;
+using TraversalHostId = std::array<char, NETPLAY_CODE_SIZE>;
+using TraversalRequestId = u64;
 
-enum TraversalPacketType
+enum class TraversalPacketType : u8
 {
   // [*->*]
-  TraversalPacketAck = 0,
+  Ack = 0,
   // [c->s]
-  TraversalPacketPing = 1,
+  Ping = 1,
   // [c->s]
-  TraversalPacketHelloFromClient = 2,
+  HelloFromClient = 2,
   // [s->c]
-  TraversalPacketHelloFromServer = 3,
+  HelloFromServer = 3,
   // [c->s] When connecting, first the client asks the central server...
-  TraversalPacketConnectPlease = 4,
+  ConnectPlease = 4,
   // [s->c] ...who asks the game host to send a UDP packet to the
   // client... (an ack implies success)
-  TraversalPacketPleaseSendPacket = 5,
+  PleaseSendPacket = 5,
   // [s->c] ...which the central server relays back to the client.
-  TraversalPacketConnectReady = 6,
+  ConnectReady = 6,
   // [s->c] Alternately, the server might not have heard of this host.
-  TraversalPacketConnectFailed = 7
+  ConnectFailed = 7
```

Comment:
Do we want to add the trailing comma while we're at it? Or i the packet type final enough so we know for sure we wont need another?
```suggestion
  ConnectFailed = 7,
```

Same for pretty much all the other enums